From: Jeremy Bicha <jbicha@ubuntu.com>
Date: Mon, 8 Oct 2018 01:29:17 -0400
Subject: skip tls_interaction test

This test is too unreliable on Debian architectures
and this package is too critical to not get timely updates

Forwarded: https://gitlab.gnome.org/GNOME/libsoup/issues/120
---
 tests/ssl-test.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tests/ssl-test.c b/tests/ssl-test.c
index 735ba41..16c4076 100644
--- a/tests/ssl-test.c
+++ b/tests/ssl-test.c
@@ -450,7 +450,6 @@ main (int argc, char **argv)
 	g_test_add_func ("/ssl/session-properties", do_session_property_tests);
 	g_test_add_func ("/ssl/message-properties/async", do_async_properties_tests);
 	g_test_add_func ("/ssl/message-properties/sync", do_sync_properties_tests);
-	g_test_add_func ("/ssl/tls-interaction", do_tls_interaction_test);
 
 	for (i = 0; i < G_N_ELEMENTS (strictness_tests); i++) {
 		g_test_add_data_func (strictness_tests[i].name,
